curve25519-dalek/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ major series.
 
 ## 4.x series
 
+### 4.1.3
+
+* Security: Fix timing leak in Scalar subtraction on u32, u64, fiat_u32, and fiat_u64 backends
```

Comment:
Should we also include various nightly/lint-related fixes like #625, #636, #638, #639, and #652?

Reply:
Yup, doing now

Reply:
Check again

Reply:
Looks good (aside from a rebase on #661)